packages/core/ts/Signal.ts

```diff
@@ -21,7 +21,7 @@ const RCTAEPSignal: ISignal = NativeModules.AEPSignal;
 const Signal: ISignal = {
   /**
    * Returns the version of the AEPSignal extension
-   * @param  {string} Promise a promise that resolves with the extension version
+   * @return  {string} Promise a promise that resolves with the extension version
```

Comment:
@return {string} Promise that resolves with the extension version

Reply:
We don't need to update it, the reason is the same as ->  https://github.com/adobe/aepsdk-react-native/pull/157#discussion_r837675134